tests/test_discounts.py

```diff
@@ -229,3 +229,21 @@ def test_sale_active(current_date, start_date, end_date, is_active):
         end_date=end_date)
     sale_is_active = Sale.objects.active(date=current_date).exists()
     assert is_active == sale_is_active
+
+@pytest.mark.parametrize('discount_value_type, discount_value, voucher_type,'
+                         'min_amount_spent, expected_min_amount_spent',[
+                        (DiscountValueType.FIXED, 2, VoucherType.PRODUCT, 100, 100),
+                        (DiscountValueType.FIXED, 2, VoucherType.COLLECTION, 100, 100),
```

Comment:
Don't need to provider `discount_value`, `min_amount_spent` and `expected_min_amount_spent` as they never change. Just hard-code them or put them into variables with the same exact names in the test body.